packages/visx-axis/src/axis/Axis.tsx

```diff
@@ -2,12 +2,13 @@ import React from 'react';
 import cx from 'classnames';
 import { Group } from '@visx/group';
 import { getTicks, coerceNumber } from '@visx/scale';
-import { SharedAxisProps, AxisScale } from '../types';
+import type { SharedAxisProps, AxisScale } from '../types';
 import AxisRenderer from './AxisRenderer';
 import getTickPosition from '../utils/getTickPosition';
 import getTickFormatter from '../utils/getTickFormatter';
 import createPoint from '../utils/createPoint';
-import Orientation, { OrientationType } from '../constants/orientation';
+import type { OrientationType } from '../constants/orientation';
+import Orientation from '../constants/orientation';
```

Comment:
FYI, instead of splitting those lines up you can also do this:

```suggestion
import Orientation, { type OrientationType } from '../constants/orientation';
```

Reply:
had just used the eslint autofix here. this doesn't seem to be configurable https://typescript-eslint.io/rules/consistent-type-imports/. no strong opinion from me.